### PR TITLE
Documentation for new WeMo Humidifier component

### PR DESCRIPTION
**Pull request by sqldiablo**
**Description:**
New component documentation page for the WeMo Humidifier component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7263

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by sqldiablo** [comment]
This is a redo of an old closed PR. I had to rework the code in homeassistant, which I just got around to. This doc page should be unchanged from my last attempt at this.

**Comment by sqldiablo** [comment]
I'm not sure why the deploy preview is failing with this error:

failed during stage 'deploying site': Invalid filename 'components/mqtt/#discovery.html'. Deployed filenames cannot contain # or ? characters

Can anyone point me in the right direction on this? This PR doesn't even touch that file.

File: source/_components/fan.wemo.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+For more configuration information, see the [WeMo component](/components/wemo/) documentation.
+
+### {% linkable_title Attributes %}
+
+There are several attributes which can be used for automations and templates.
+
+| Attribute | Description |
+| --------- | ----------- |
+| `current_humidity` | An integer that indicates the current relative humidty percentage of the room, as determined by the device's onboard humidity sensor.
```
Comment (MartinHjelmare) [inline source/_components/fan.wemo.markdown]: ...the current relative humidity percentage...

File: source/_components/fan.wemo.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+| `target_humidity` | An integer that indicates the desired relative humidity percentage (this is constrained to the humidity settings of the device, which are 45, 50, 55, 60, and 100).
+| `fan_mode` | String that indicates the current fan speed setting (Off, Low, Medium, or High). The WeMo humidifier speeds of minimum and low both map to the Home Assistant speed of low, and the WeMo humidifier speeds of high and maximum both map to the Home Assistant speed of high (due to constraints on which speeds Home Assistant supports for fans)
```
Comment (MartinHjelmare) [inline source/_components/fan.wemo.markdown]: Missing period at the end of the section.

**Review by MartinHjelmare — APPROVED** [review]

**Comment by MartinHjelmare** [comment]
There's a build error. Rebasing on fresh next could help.

**Comment by sqldiablo** [comment]
I'm going to have to close this PR and delete my fork to fix this, like I had to do for my fork of home-assistant as well.

**Pull request closed without merging**